packages/core/src/bodl/bodl-emitter-service.ts

```diff
@@ -96,8 +102,10 @@ export default class BodlEmitterService implements BodlService {
     private getProducts(lineItems: LineItemMap, currencyCode: string): BODLProduct[] {
         const customItems: BODLProduct[] = (lineItems.customItems || []).map(item => ({
             product_id: item.id,
-            product_sku: item.sku,
-            price: item.listPrice,
+            sku: item.sku,
+            base_price: item.listPrice,
+            sale_price: item.listPrice,
```

Comment:
Item also has a `salePrice` property, should we be using that?

Reply:
it's pretty much the same as below